source/index.txt

```diff
@@ -9,6 +9,7 @@
    Get Started </get-started>
    /connect
    /databases-collections
+   /run-command
```

Comment:
[note to reviewer] This will move to the `/databases-collections` section once [DOCSP-4063](https://github.com/mongodb/docs-c/pull/55) is merged.